Makefile

```diff
@@ -11,4 +11,6 @@ release:
 
 
 build-docs:
+	# pip install -r requirements-doc.txt
+	# conda install -c conda-forge pymc
```

Comment:
Is this needed ? If so why is it commented ?

Reply:
Remove `conda install -c conda-forge pymc` since `use_with_pymc.py` is removed from documentation. also uncommented dependencies installation.